app/Policies/MemberPolicy.php

```diff
@@ -100,6 +102,10 @@ public function listOldMembers(User $user)
         return $user->hasCapability("members::old::show");
     }
 
+    public function showPhone(User $user, Member $member) {
+        return $member->hasRole("counselor");
+    }
+
```

Comment:
we need an extra right for if the user has showPrivate rights or if you are the member itself